src/Install/Installer.php

```diff
@@ -299,12 +299,18 @@ private function installSaferPayCardAlias()
     private function installSaferPayLog()
     {
         return Db::getInstance()->execute(
-            'CREATE TABLE IF NOT EXISTS ' . _DB_PREFIX_ . 'saferpay_log' . '(
-            `id_saferpay_log` INTEGER(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
-            `message` TEXT NOT NULL,
-            `payload` TEXT NOT NULL,
-            `date_add` datetime NOT NULL
-                ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci'
+            'CREATE TABLE IF NOT EXISTS ' . _DB_PREFIX_ . pSQL(\SaferPayLog::$definition['table']) . '(
+                `id_saferpay_log` INTEGER(10) UNSIGNED AUTO_INCREMENT,
+                `id_log` INT(10) DEFAULT 0,
+                `id_shop` INT(10) DEFAULT ' . Configuration::get('PS_SHOP_DEFAULT') . ',
```

Comment:
Configuration::get('PS_SHOP_DEFAULT') might return string make sure to cast it to (int)